src/github/pull-request.ts

```diff
@@ -111,3 +95,29 @@ export default async (
 		jiraResponse?.status
 	);
 };
+
+const updateGithubIssues = async (github: GitHubClient | GitHubAPI, context: CustomContext, util, pullRequest) => {
```

Comment:
@mboudreau  - this seems to do similar as https://github.com/atlassian/github-for-jira/pull/1117/files, I am considering re-using your issues method or moving them both to a generic location?

Reply:
yep, sounds like a great idea.